pkg/webhook/indexeres/indexer.go

```diff
@@ -0,0 +1,23 @@
+package indexeres
+
+import (
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/webhook/clients"
+)
+
+const (
+	VMBackupBySourceUIDIndex = "harvesterhci.io/vmbackup-by-source-uid"
+)
+
+func RegisterIndexers(clients *clients.Clients) error {
+	vmBackupCache := clients.HarvesterFactory.Harvesterhci().V1beta1().VirtualMachineBackup().Cache()
+	vmBackupCache.AddIndexer(VMBackupBySourceUIDIndex, vmBackupBySourceUID)
```

Comment:
could `AddIndexer ` be failed and return an error?

Reply:
No, I remove the return error. Thank you.